### PR TITLE
1.17.1 fabric

### PR DESCRIPTION
**Pull request by Curodenz**
porting Jade to Fabric 1.17.1

### CODE REVIEW

File: src/main/java/snownee/jade/util/LinkedTreeMap.java
Comment (Snownee) [inline src/main/java/snownee/jade/util/LinkedTreeMap.java:1]: What's this?

**Comment by Curodenz** [comment]
您好，非常感谢您的邮件！我已经确实地收到邮件了，还请稍稍耐心等待我的回复。Hello, thank you very much for your email!I have indeed received the email. Please be patient and wait for my reply.

**Comment by Snownee** [comment]
It looks fine, but you copied your code directly to the default branch instead of the branch you were working on, which makes it really hard for me to tell what exactly you changed.

**Comment by Snownee** [comment]
You also didn't delete the new classes, for example CommonProxy

File: src/main/java/snownee/jade/util/LinkedTreeMap.java
Comment (Curodenz) [reply src/main/java/snownee/jade/util/LinkedTreeMap.java:1]: I think this is a class left over from the porting process to try to implement some necessary functions in newer version of gson with gson 2.8.0, and I forgot to remove it.

**Comment by Curodenz** [comment]
> You also didn't delete the new classes, for example CommonProxy
Sorry I overlooked, I'll rearrange the project file and submit it

**Comment by Curodenz** [comment]
> 

Sorry I overlooked, I'll rearrange the project file and submit it again

**Comment by Snownee** [comment]
Please open a new PR when you are done.

**Pull request closed without merging**